content/influxdb/v2.0/security/tokens/use-tokens.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Use tokens
+seotitle: Use an authentication token in InfluxDB
+description: Use an authentication token in InfluxDB using the InfluxDB UI, `influx` CLI, or InfluxDB API.
+aliases:
+  - /influxdb/v2.0/users/tokens/use-tokens
+menu:
+  influxdb_2_0:
+    name: Use tokens
+    parent: Manage tokens
+weight: 204
+---
+
+Use tokens to authenticate requests to InfluxDB, including requests to write, query, and manage data and resources.
+Authenticate requests using the [`influx` CLI](/influxdb/v2.0/reference/cli/influx/), API requests made with client libraries, and tools like `curl`.
+
+## Add a token to a CLI request
+
+### Example: token on command line
```

Comment:
##### H5s for examples

what do you think about dropping the "Example" heading altogether? ...To avoid doubling up headers w/o text in between. Folks'll get a hit on "add a token to a CLI request" and see both examples

Reply:
Done 👍